doc/sphinx/source/tutorials/run-fit.rst

```diff
@@ -3,32 +3,42 @@
 How to run a PDF fit
 ====================
 
-
 The user should perform the steps documented below in order to obtain a complete
 PDF fit using the latest release of the NNPDF fitting code: ``n3fit``.
-The fitting methodology is detailed in the [Methodology](methodology) page.
+The fitting methodology is detailed in :ref:`methodology`.
+
+The three main points in this tutorial are:
 
 - :ref:`Preparing a fit runcard <prepare-fits>`
 - :ref:`Running the fitting code <run-n3fit-fit>`
 - :ref:`Upload and analyse the fit <upload-fit>`
+- :ref:`advance-run-fit`
 
 
 .. _prepare-fits:
+
 Preparing a fit runcard
 -----------------------
 
 The runcard is written in YAML. The runcard is the unique identifier of a fit
-and contains all required information to perform a fit, which includes the
+and contains all required information to perform and reproduce a fit, which includes the
 experimental data, the theory setup and the fitting setup.
-
 A detailed explanation on the parameters accepted by the ``n3fit`` runcards
-can be found in the :ref:`detailed guide <runcard-detailed>`.
+can be found in the :ref:`n3fit detailed guide <runcard-detailed>`.
 
 For newcomers, it is recommended to start from an already existing runcard,
 example runcards (and runcard used in NNPDF releases) are available at
 `n3fit/runcards <https://github.com/NNPDF/nnpdf/tree/master/n3fit/runcards>`_.
+
+.. note::
+
+  While we aim for the code to be both backwards and forwards compatible with respect to runcards,
+  by setting sensible defaults when introducing new features,
+  make sure that you are using a runcard tagged with the same version of the code you are using to
+  avoid any surprises.
```

Comment:
```suggestion
  make sure that you are using a runcard tagged with the same version of the code you are using to
  avoid any surprises.
```

If by tagged version you mean the example runcards as corresponding to a given commit, I don't think this is a very clear explanation.